chameleon/settings.py

```diff
@@ -639,3 +642,12 @@
 CELERY_RESULT_SERIALIZER = 'json'
 CELERY_TASK_SERIALIZER = 'json'
 OUTAGE_EMAIL_REMINDER_TIMEDELTA = (60 * 60 * 2)
+OUTAGE_REMINDER_FREQUENCY = 30
```

Comment:
I'd prefer we keep all of these in the same unit - can you make this seconds? You can keep the cron schedule in minutes, just divide by 60 (and cast to `int`!). It makes it easier to compare the deltas w/ the frequency when examining the config. No surprises.